TeachMeSkillsDomashneeZadanie

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
You need to create a java class with main method that prints out smth like "Hello".

And please create a separate module for this class (you should name it Lesson1)